Source/EasyNetQ/DI/ConnectionFactoryFactory.cs

```diff
@@ -23,7 +23,8 @@ public static IConnectionFactory CreateConnectionFactory(ConnectionConfiguration
                 NetworkRecoveryInterval = configuration.ConnectIntervalAttempt,
                 ContinuationTimeout = configuration.Timeout,
                 DispatchConsumersAsync = true,
-                ConsumerDispatchConcurrency = configuration.PrefetchCount
+                ConsumerDispatchConcurrency = configuration.PrefetchCount,
+                RequestedChannelMax = configuration.RequestedChannelMax
```

Comment:
do we like this name?

Reply:
I took it from the [RabbitMQ client](https://www.rabbitmq.com/channels.html)

Reply:
So we love it 😅

Reply:
Didn't saw that this was the creation of the rabbitmq client factory, what I don't understand is that by default they should use the server max channel so it should not fail silently as we supposed, BTW the PR is good to configure that value to a lower value, but I'm not sure on the models part, does we really need that part? Need more time to focus on it

Reply:
Ok your need is to fix the max number of channel per connection and avoid easynetq create more than the specified

Reply:
This part is just an additional option to limit the channels per connection (for me to one). Maybe for others to a low value to reduce memory load on IoT devices. The default channel limit is set to the server default (~2000).